kotlinx.fuzz.api/build.gradle.kts

```diff
@@ -1,12 +1,15 @@
 plugins {
     id("kotlinx.fuzz.src-module")
+    `kotlin-dsl`
 }
 
 dependencies {
     implementation(libs.rgxgen)
     implementation(libs.junit.platform.engine)
     implementation(kotlin("reflect"))
 
+    implementation(libs.junit.platform.engine)
```

Comment:
Duplicate: same dependency defined on the line 8

Reply:
Removed